src/lwc/gsApplicationStatus/gsApplicationStatus.html

```diff
@@ -0,0 +1,37 @@
+<template >
+    <template if:false={isActiveInstance}>
+        <lightning-card>
+            <div class="spinner" if:true={isLoading}>
+                <lightning-spinner alternative-text="Loading"></lightning-spinner>
+            </div>
+            <div class="slds-grid slds-grid_align-center slds-grid_vertical">
+                <div class="slds-col slds-align_absolute-center">
+                    <img class="image" src={img}/>
+                </div>
+                <div if:false={isApplicationSubmitted} class="slds-col slds-align_absolute-center slds-size_full">
+                    <div class="slds-text-align_center slds-size_full slds-p-around_small">
+                        <div><p class="slds-text-heading_small slds-m-bottom_medium">{labels.gsNoApplicationSubmitted} | <a tabindex="0">{labels.gsLearnMore}</a></p></div>
+                        <div><p class="slds-text-heading_small slds-m-bottom_medium daysLeft">{diffInDays} {labels.gsDaysRemainingInFreeTrial}</p></div>
+                        <div><a tabindex="0" class="slds-button slds-button_brand slds-size_full slds-text-heading_small slds-p-around_medium" href="https://powerofus.force.com/PowerOfUsApply" target="_blank">{labels.gsApplyForFreeLicenses}</a></div>
+                    </div>
+                </div>
+                <div if:true={isApplicationSubmitted} class="slds-col slds-align_absolute-center slds-size_full">
+                    <div class="slds-text-align_center slds-size_full slds-p-around_small">
+                        <div><p class="slds-text-heading_small slds-m-bottom_medium">{labels.gsApplicationStatus}<span class="slds-text-color_success">{labels.gsSubmitted}</span></p></div>
+                        <div><p class="slds-text-heading_small slds-m-bottom_medium daysLeft">{diffInDays} {labels.gsDaysRemainingInFreeTrial}</p></div>
+                        <div><button onclick={onCheckStatusClick}
```

Comment:
lightning-button?

Reply:
Please use `lightning-button` and just put `{labels.gsCheckStatus}` into the `name` attribute.

Reply:
I can't on this one. The button design is not a standard salesforce lightning-button like. All the variants make the button look different than the design